engine/graphics/src/graphics_glfw_wrappers.cpp

```diff
@@ -12,25 +12,71 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
-#include <glfw/glfw.h>
-#include  <glfw/glfw_native.h>
 #include "graphics_native.h"
 
+#include "graphics_private.h"
+
+#if defined(DM_PLATFORM_IOS)
+    #include <platform/platform_window_ios.h>
+#elif defined(ANDROID)
+    #include <platform/platform_window_android.h>
+#elif defined(__MACH__)
+    #include <platform/platform_window_osx.h>
+#elif defined(_MSC_VER)
+    #include <platform/platform_window_win32.h>
+#elif defined(__linux__) && !defined(ANDROID)
+    #include <platform/platform_window_linux.h>
+#endif
```

Comment:
Do we expose DM_PLATFORM_ defines for all platforms? I couldn't get it to work properly on os 🤔

Reply:
According to waf_dynamo, no. There we only do ios/macos.
On the extender server though (build_input.yml) we do.